DEVELOPER_STARTING_GUIDE.md

```diff
@@ -0,0 +1,185 @@
+
+- Have a look at the [demo project](https://solid-design-system.github.io/solid-design-system-demo/) to get a better understanding of the design system and its components. You can even [install it locally and play around](https://github.com/solid-design-system/solid-design-system-demo) to explore its features and functionalities.
+
+- Solid Components follows a monorepo structure with packages (e. g. `components`) managed by `pnpm` (which is a replacement for `npm`). Linting and Formatting is centralized at root level. Packages have to be run individually (e.g. `cd packages/components && pnpm dev` to start development server). Run `pnpm verify` at the root directory periodically, particularly, before pushing changes when a pull request is already opened.
+
+## SOLID Library Concepts
+
+### Web **Components** & CSS **Styles**
+
+We are creating a library that comprises a lean combination of reusable [Web Components](https://developer.mozilla.org/en-US/docs/Web/API/Web_components) and [BEM format](https://css-tricks.com/bem-101/) CSS Classes that adhere to [Union design standards](https://www.figma.com/files/1075429990769806468/project/67503549/Solid-DS-Documentation?fuid=883643809929820461). Each have their own technology and purpose:
+
+- **Styles (aka style-components)**: These are standalone CSS files. They don't provide any logic and are often non-interactive.
+- **Components**: These are Web Components. They are often more complex than Styles, and could feature reactivity, state, multiple slots, properties, and more. They can be used in any framework.
+  - [slots](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/slot) allow for easy element nesting ([Lit: Working with Shadow DOM](https://lit.dev/docs/components/shadow-dom/))
+  - [parts](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/part) allow external manipulation of CSS.
+  - [events](https://lit.dev/docs/components/events/) make critical information available outside of a component.
+
+### Samples & Patterns
+
+Alongside the documentation for Styles and Components, we also present Samples and Patterns. These differ in their scope:
+
+- **Samples**: Tied to a specific component, these show its use in a particular context. They are added as "Sample" stories in the component's documentation.
+- **Patterns**: More advanced, they demonstrate how to combine several components to solve a specific problem. They are added as stories in the "Patterns" section.
+
+## Shoelace
+
+The [Shoelace](https://shoelace.style/) web component library is our working inspiration. Components are copied directly from their [source code](https://github.com/shoelace-style/shoelace) and pasted in the `packages/components/src/_components` folder.
+
+_There may have been updates since the files there were copied to our repo!_
```

Comment:
This statement might need some followup instructions. New devs might not infer that they should copy the components from the Shoelace repo in these cases instead of relying on the ones in _components.

Something along the lines of "In case of updates, copy the source code from the latest version of the Shoelace repository."

Reply:
Get your point. 
Any suggestions?

Reply:
Added a suggestion below so closing this to avoid redundancy.